### PR TITLE
Add WProofreader to feature list

### LINKED ISSUE #1583: [Docs] Add WSC's proofreader to the Features section
WebSpellChecker created a WProofreader solution for CKEditor 5: https://webspellchecker.com/wsc-proofreader/#proofreader-ckeditor5

Let's add it to the "Features" section of the documentation.

### LINKED ISSUE #1916: Add spell checker feature (WProofreader) to Features documentation
We will be launching the new spell checking solutions product page on our website soon, so it's time to add a dedicated Features page for WProofreader to our documentation.

Their demo of CKEditor 5 integration is here: https://webspellchecker.com/wsc-proofreader/#proofreader-ckeditor5

### PR DESCRIPTION
**Pull request by msamsel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Add WProofreader to feature list. Closes #1583.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

**Comment by mlewand** [comment]
@AnnaTomanek please give a hand to @msamsel with this task. I have requested WSC team to provide us with a key that we should use for this demo purpose.

**Comment by mlewand** [comment]
BTW the PR should be pointed at `stable`.

**Comment by msamsel** [comment]
I've rebased branch to `stable`

File: docs/_snippets/features/wproofreader.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ClassicEditor
+	.create( document.querySelector( '#snippet-wproofreader' ), {
+		cloudServices: CS_CONFIG,
+		toolbar: {
+			viewportTopOffset: window.getViewportTopOffsetConfig()
+		}
+	} )
+	.then( editor => {
+		window.editor = editor;
+		// Temporary fix for WProofreader dialog.
```
Comment (mlewand) [inline docs/_snippets/features/wproofreader.js]: Please, add a reference to this issue: https://github.com/WebSpellChecker/wproofreader/issues/17

**Pull request merged**